### DISCUSSION TITLE
How to make the result of the function rfsrc stable？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#240] [Q&A] [answered]
library(randomForestSRC)
set.seed(11)
iris.obj <- rfsrc(Species ~., data = iris,  importance =  "anti")
plot(iris.obj)

**Run twice：**
![image](https://user-images.githubusercontent.com/5112397/166891546-1a80a988-b6e5-41fe-8dfc-128e9e49cace.png)
![image](https://user-images.githubusercontent.com/5112397/166891601-f28a8843-49a2-468f-9468-f836831a994e.png)
**Even though we set up random seeds, the results are still different.**
**How to make the result of the function rfsrc stable？**

**Comment by A-Pai** [comment]
library(randomForestSRC)
iris.obj <- rfsrc(Species ~., data = iris,  importance =  "anti", seed = 11)
plot(iris.obj)

**---That doesn't work either**

**Comment by ishwaran** [comment, marked as answer]
As we answered in your previous question, reproducibility exists only in limited conditions.  Please **READ** the following

https://github.com/kogalur/randomForestSRC/issues/67#issuecomment-1100196455
